examples/chain_client/exchange/14_MsgCreateBinaryOptionsLimitOrder.py

```diff
@@ -58,7 +47,6 @@ async def main() -> None:
         margin=Decimal("0.5"),
```

Comment:
Why in some places we leave `Decimal` and in some not?

Reply:
Sorry @PavelInjective. What do you mean with "we leave Decimal"?
Or maybe you can mention an example of those places you mean we don't leave Decimal